Packs/MicrosoftExchangeOnline/Integrations/EWSO365/README.md

```diff
@@ -1,147 +1,143 @@
-    *   **Use a self deployed Azure Application**: Select this checkbox if you are using a self-deployed Azure application.
-
-4. Azure cloud options
-
-    | Azure Cloud | Description                                                              |
-    |-------------|--------------------------------------------------------------------------|
-    | Worldwide   | The publicly accessible Azure Cloud                                      |
-    | US GCC      | Azure cloud for the USA Government Cloud Community (GCC)                 |
-    | US GCC-High | Azure cloud for the USA Government Cloud Community High (GCC-High)       |
-    | DoD         | Azure cloud for the USA Department of Defense (DoD)                      |
-    | Germany     | Azure cloud for the German Government                                    |
-    | China       | Azure cloud for the Chinese Government                                   |
-
-5. Click **Test** to validate the URLs, token, and connection.
+    You can delete the items permanently (hard delete) or delete the items (soft delete), so they can be recovered by running the `ews-recover-messages` command.
+
+## Architecture
+
+This integration is based on the `exchangelib` python module. For more information about the module, check the [documentation](https://ecederstrand.github.io/exchangelib/).
+
+
+## Set up the Third Party System
+
+There are two application authentication methods available.
+Follow your preferred method's guide on how to use the admin consent flow in order to receive your authentication information:
+
+* [Cortex XSOAR Application](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#cortex-xsoar-application)
+* [Self-Deployed Application](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#self-deployed-application) - Client Credential Flow.
 
 ## Authentication
 
 For more details about the authentication used in this integration, see [Microsoft Integrations - Authentication](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication).
 
-### Required Permissions for self deployed Azure Applications
+## Permissions
+
+In order to function as expected, the service account should have:
+
+**Impersonation rights** - In order to perform actions on the target mailbox of other users, the _service account_ must be part of the `ApplicationImpersonation` role. For more information and instructions on how to set up the permission, see [Microsoft Documentation](https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/impersonation-and-ews-in-exchange).
+Most command require this permission to function correctly. This permission is specified in each relevant command's Permission section. For more information, see [Microsoft Documentation](https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/impersonation-and-ews-in-exchange). 
```

Comment:
```suggestion
Most commands require this permission to function correctly. This permission is specified in each relevant command's Permission section. For more information, see [Microsoft Documentation](https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/impersonation-and-ews-in-exchange). 
```

Reply:
also. is this intended: [Microsoft Documentation] ?

Reply:
Yes it is a link to the docs.